orangecanvas/application/utils/addons.py

```diff
@@ -290,13 +293,15 @@ def _session(cachedir=None):
     -------
     session : requests.Session
     """
-    import cachecontrol.caches
     if cachedir is None:
         cachedir = QStandardPaths.writableLocation(QStandardPaths.CacheLocation)
-        cachedir = os.path.join(cachedir, "networkcache", "requests")
-    session = requests.Session()
-    session = cachecontrol.CacheControl(
-        session, cache=cachecontrol.caches.FileCache(directory=cachedir)
+        cachedir = os.path.join(cachedir, "networkcache")
+    session = requests_cache.CachedSession(
+        os.path.join(cachedir, "requests.sqlite"),
+        backend=requests_cache.SQLiteCache,
```

Comment:
In my case, providing class doesn't work. I think, according to documentation, it should be either string (`sqlite`) or an already initiated object (`requests_cache.SQLiteCache()`).

```suggestion
        backend="sqlite",
```

Reply:
Otherwise it seems to be working well